ledger/metrics/src/main/scala/com/daml/metrics/UpdateEventsMetrics.scala

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics
+
+import com.daml.metrics.api.MetricDoc.MetricQualification.Debug
+import com.daml.metrics.api.MetricHandle.Factory
+import com.daml.metrics.api.{MetricDoc, MetricHandle, MetricName}
+
+class UpdateEventsMetrics(prefix: MetricName, metricFactory: Factory) {
```

Comment:
'Updates' represent 'events' received by the participant, hence, I'd suggest renaming this to `ReceivedUpdates` or `IndexedUpdates` (as it is under the `daml.metrics` package), or to `EventMetrics` or `ReceivedEventsMetrics`.